crates/ra_syntax/tests/data/parser/inline/ok/0055_literal_pattern.txt

```diff
@@ -8,22 +8,23 @@ SOURCE_FILE@[0; 95)
       L_PAREN@[7; 8)
       R_PAREN@[8; 9)
     WHITESPACE@[9; 10)
-    BLOCK@[10; 94)
+    BLOCK@[10; 112)
       L_CURLY@[10; 11)
       WHITESPACE@[11; 16)
-      MATCH_EXPR@[16; 92)
+      MATCH_EXPR@[16; 110)
         MATCH_KW@[16; 21)
         WHITESPACE@[21; 22)
         TUPLE_EXPR@[22; 24)
           L_PAREN@[22; 23)
           R_PAREN@[23; 24)
         WHITESPACE@[24; 25)
-        MATCH_ARM_LIST@[25; 92)
+        MATCH_ARM_LIST@[25; 110)
           L_CURLY@[25; 26)
           WHITESPACE@[26; 35)
           MATCH_ARM@[35; 43)
-            LITERAL@[35; 37)
-              INT_NUMBER@[35; 37) "92"
+            MINUS@[35; 36)
+            LITERAL@[36; 37)
+              INT_NUMBER@[36; 37) "1"
```

Comment:
the reference writes [negative number is not literal](https://doc.rust-lang.org/beta/reference/patterns.html#literal-patterns), that's why I "split" MINUS from `-1`, is it ok?

Reply:
:ok_hand: